ods_ci/tests/Tests/100__deploy/120__upgrades/122__post_ugrade.robot

```diff
@@ -126,8 +127,27 @@ Set Default Users
     [Documentation]  Set Default user settings
     Set Standard RHODS Groups Variables
     Set Default Access Groups Settings
-
+    Upgrade Test Teardown
```

Comment:
could you re-use the same one from `ods_ci/tests/Tests/100__deploy/120__upgrades/121__during_upgrade.robot`?

Reply:
Nope you can not use keyword from the suitre whihc has TC and keyword

Reply:
of course, I meant to say you can put that kw in a common resource file and reuse it